airflow/configuration.py

```diff
@@ -342,10 +342,6 @@ def _get_option_from_default_config(self, section, key, **kwargs):
                 self.airflow_defaults.get(section, key, **kwargs))
 
         else:
-            log.warning(
```

Comment:
I think this one should be kept - if we provide a `fallback` when getting a config this won't show up (I think), so seeing this warning is a clue to the user that they need to configure something

Reply:
I thought that raising `AirflowConfigException` below will be enough

Reply:
Oh I forgot about that :+1: